areas/servicebus/tests/AzureMcp.ServiceBus.UnitTests/Queue/QueuePeekCommandTests.cs

```diff
@@ -0,0 +1,326 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
I would unstage these since we don't expose the peek command.

Reply:
I deleted the peek commands tests and the respective json converter since the command is not exposed to public. In case we decide to add these commands again, we can always add back the latest commit in this PR.